shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -1093,10 +1093,13 @@ public void onFlutterTextureViewCreated(@NonNull FlutterTextureView flutterTextu
   @Override
   public void onFlutterUiDisplayed() {
     // Notifies Android that we're fully drawn so that performance metrics can be collected by
-    // Flutter performance tests.
-    // This was supported in KitKat (API 19), but has a bug around requiring
-    // permissions. See https://github.com/flutter/flutter/issues/46172
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+    // Flutter performance tests. A few considerations:
+    // * rFD was supported in KitKat (API 19), but has a bug around requiring
+    // permissions in some Android versions.
+    // * rFD behavior isn't tested on pre-Q versions.
+    // See https://github.com/flutter/flutter/issues/46172, and
+    // https://github.com/flutter/flutter/issues/88767.
```

Comment:
nit: consider adding that there are vendor-specific bugs.

How confident are we that this API works on all vendors after Q?

Reply:
I only know that a CTS test was added. Vendors should be running these tests, but we don't really know.

Reply:
SGTM